tests/jerry/json-stringify.js

```diff
@@ -212,3 +212,7 @@ assert (JSON.stringify (object, null, undefined) == '{"a":2}');
 assert (JSON.stringify (object, null, new Boolean (true)) == '{"a":2}');
 assert (JSON.stringify (object, null, [1, 2, 3] ) == '{"a":2}');
 assert (JSON.stringify (object, null, { "a": 3 }) == '{"a":2}');
+
+// Test with proxy
+assert(JSON.stringify(new Proxy(['foo'], {})) === '["foo"]');
```

Comment:
Proxy is only supported in case of es2015 profile. This test should be moved into the "es2015" directory.